datafusion/functions/src/encoding/inner.rs

```diff
@@ -553,3 +570,29 @@ fn decode(args: &[ColumnarValue]) -> Result<ColumnarValue> {
     }?;
     decode_process(expression, encoding)
 }
+
+#[cfg(test)]
+mod tests {
+    #[test]
+    fn test_encode_fsb() {
+        use super::*;
+
+        let value = vec![0u8; 16];
+        let array = arrow::array::FixedSizeBinaryArray::try_from_sparse_iter_with_size(
+            vec![Some(value)].into_iter(),
+            16,
+        )
+        .unwrap();
+        let value = ColumnarValue::Array(Arc::new(array));
+
+        let ColumnarValue::Array(result) =
+            encode_process(&value, Encoding::Base64).unwrap()
+        else {
+            panic!("unexpected value");
+        };
```

Comment:
To cover the missing usage of `return_type` and `coerce_types` you need something like:
```rust
let encode_func = EncodeFunc::new();
    let args = vec![
        ColumnarValue::Array(Arc::new(array)),
        ColumnarValue::Scalar(ScalarValue::Utf8(Some("base64".to_string()))),
    ];
    
    // This will test the full path including type checking
    let result = encode_func.invoke_with_args(datafusion_expr::ScalarFunctionArgs {
        args,
        number_rows: 1,
        return_type: &DataType::Utf8,
    }).unwrap();
    
    let ColumnarValue::Array(result) = result else {
        panic!("unexpected value");
    };
    ...
```